Makefile

```diff
@@ -1,2 +1,5 @@
-uthread: main.cpp
-	g++ --std=c++11 -o uthread main.cpp -lrt
+uthread: main.cpp uthread.cpp uthread.h
+	g++ --std=c++11 -o uthread main.cpp uthread.h uthread.cpp -lrt
+
+mergesort: uthread.cpp merge_sort.cpp uthread.h
```

Comment:
the file is called mergesort.cpp,  not merge_sort.cpp (no underscore)

Reply:
I added it as `merge_sort.cpp`.